README.md

```diff
@@ -1,12 +1,12 @@
-# Podman Desktop
-
-Install, configure and keep container engine (Podman) up to date. System tray to keep it running and switch Kubernetes contexts. 
-Desktop App to interact with containers and pods running in Podman. Connect and deploy Pods to Kubernetes/OpenShift environments.
+Podman Desktop - A graphical tool for developing containers locally and on Kubernetes
```

Comment:
I don't think the tag line here is right. 

We previously define this one: "a GUI for Containers and Kubernetes for application developers". We can surely refine/iterate on it.

Do we need to repeat the name of the tool in the title? Should we go straight into something like "Containers and Kubernetes for application developers"